Dolphin scripts/Entrance Randomizer/lib/constants.py

```diff
@@ -156,27 +156,68 @@ class Addresses:
 
 
 class LevelCRC(IntEnum):
-    JAGUAR = 0x99885996
+    ABANDONED_CAVERN = 0xEA667977
+    ALTAR_OF_AGES = 0xABD7CCD8
+    ALTAR_OF_HUITACA = 0x70EBFCA3
+    APU_ILLAPU_SHRINE = 0x5511C46C
+    BATTERED_BRIDGE = 0x69F0CDE2
+    BETA_VOLCANO = 0x47508760
+    BITTENBINDERS_CAMP = 0x0EF63551
+    BUTTERFLY_GLADE = 0xCD944049
+    CAVERN_LAKE = 0x8B372E42
+    CHAMELEON_TEMPLE = 0x0081082C
+    COPACANTI_LAKE = 0x8147EA91
     CRASH_SITE = 0xEE8F6900
-    PLANE_COCKPIT = 0x4A3E4058
+    CRYSTAL_CAVERN = 0x0DDE5470
+    EKKEKO_ICE_CAVERN = 0x3E7EE822
+    EYES_OF_DOOM = 0x9A0C8DF8
+    FIRE_BOMBED_TOWERS = 0xF0F99C58
+    FLOODED_CAVE = 0x3A811024
     FLOODED_COURTYARD = 0xECC9D759
-    CHAMELEON_TEMPLE = 0x0081082C
+    GATES_OF_EL_DORADO = 0x93F89D45
+    GREAT_TREE = 0x3292B6C9
+    JAGUAR = 0x99885996
     JUNGLE_CANYON = 0xDEDA69BC
+    JUNGLE_TRAIL = 0x7A9B3870
+    KABOOM = 0xE411440A
+    LADDER_OF_MILES = 0x619E1126
     MAMA_OULLO_TOWER = 0x07ECCC35
-    VIRACOCHA_MONOLITHS = 0x6F498BBD
-    VIRACOCHA_MONOLITHS_CUTSCENE = 0xE8362F5F
-    ALTAR_OF_AGES = 0xABD7CCD8
-    BITTENBINDERS_CAMP = 0x0EF63551
+    MONKEY_SPIRIT = 0x02C7B675
+    MONKEY_TEMPLE = 0xF3B4DC8E
+    MOUNTAIN_OVERLOOK = 0x907C492B
+    MOUNTAIN_SLED_RUN = 0x1B8833D3
+    MOUTH_OF_INTI = 0xB8D5CE86
     MYSTERIOUS_TEMPLE = 0x099BF148
-    EYES_OF_DOOM = 0x9A0C8DF8
-    SCORPION_TEMPLE = 0x4B08BBEB
+    NATIVE_JUNGLE = 0x0AF1CCFF
+    NATIVE_VILLAGE = 0x05AA726C
+    PENGUIN_DEN = 0x1553BBE1
+    PENGUIN_SPIRIT = 0x1F237F32
     PENGUIN_TEMPLE = 0x1B11EC74
-    VALLEY_OF_THE_SPIRITS = 0x08E3C641
-    COPACANTI_LAKE = 0x8147EA91
-    APU_ILLAPU_SHRINE = 0x5511C46C
+    PICKAXE_RACE = 0x7A75D1A9
+    PLANE_COCKPIT = 0x4A3E4058
+    PLANE_CUTSCENE = 0x53257119
+    PUNCHAU_SHRINE = 0x38C7AE7D
+    PUSCA = 0xCFD2FE10
+    RAFT_BOWLING = 0x9316749C
+    RENEGADE_FORT = 0x0C1C3E47
+    RENEGADE_HEADQUARTERS = 0x1CB1432D
+    SCORPION_SPIRIT = 0x0305DC42
+    SCORPION_TEMPLE = 0x4B08BBEB
+    SPINJA_LAIR = 0x7652BAFC
+    ST_CLAIRE_DAY = 0xBA9370DF
     ST_CLAIRE_NIGHT = 0x72AD42FA
-    ST_CLAIRE_DAY = 0x72AD42FA
+    STATUES_OF_AYAR = 0xA85A2793
     TELEPORTS = 0xE97CB47C
+    TUCO_SHOOT = 0x0D72E13F
+    TURTLE_MONUMENT = 0x239A2165
+    TWIN_OUTPOSTS = 0xE6B9138A
+    TWIN_OUTPOSTS_UNDERWATER = 0xDE524DA6
+    UNDERGROUND_DAM = 0x9D6149E1
+    VALLEY_OF_THE_SPIRITS = 0x08E3C641
+    VIRACOCHA_MONOLITHS = 0x6F498BBD
+    VIRACOCHA_MONOLITHS_CUTSCENE = 0xE8362F5F
+    WHACK_A_TUCO = 0x0A1F2526
+    WHITE_VALLEY = 0x62548B77
```

Comment:
Thanks! Btw changes like these are very easy to split into their self contained PRs. If you split that off I can merge it basically instantly.